modules/javafx.graphics/src/test/java/test/javafx/scene/CssStyleHelperTest.java

```diff
@@ -29,6 +29,8 @@
 import com.sun.javafx.tk.Toolkit;
 import java.io.IOException;
 import javafx.css.CssParser;
+import javafx.css.CssParser.ParseError;
+import javafx.css.CssParser.ParseError.PropertySetError;
```

Comment:
very minor: I would have used `ParseError.PropertySetError` in the code instead of static import, removes the guesswork when looking at the code.